rudder-agent/debian/postinst

```diff
@@ -32,7 +32,7 @@ case "$1" in
 
     CFRUDDER_USE_SYSTEMD="false"
 
-    if readlink -f $(cut -f 1 -d " " < /proc/1/cmdline) | grep -qE "/systemd$" ; then CFRUDDER_USE_SYSTEMD="true"; fi
+    if readlink -f $(strings /proc/1/cmdline | head -n1 | cut -f 1 -d " ") | grep -qE "/systemd$" ; then CFRUDDER_USE_SYSTEMD="true"; fi
```

Comment:
this requires to add binutils as dependency for the agent

Reply:
it is not installed by default in base docker images, and probably some stripped-down VMs too.